less/matching.less

```diff
@@ -112,8 +113,7 @@
 			.matching-select-icon {
 				display: none;
 			}
-			.incorrect .matching-correct-icon,
-			.correct .matching-correct-icon {
+			.matching-correct-icon {
```

Comment:
Is there a reason this selector has been modified? This will break the `_canShowMarking` functionality introduced in https://github.com/adaptlearning/adapt-contrib-matching/pull/93.